packages/react-native-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -172,6 +178,10 @@ public boolean isContentChanged() {
         void onQueryCurrentProgressForSavingMedia();
     }
 
+    /* public interface OnSetFeaturedImageListener {
```

Comment:
Hey Siobhan, aha, can you elaborate on why this new interface is commented out and the new featured image related interface functions are placed in `OnMediaLibraryButtonListener`? I'm not sure if `OnMediaLibraryButtonListener` is actually related 🤔, I'd expect this new/separate interface definition to be the desired implementation.

Reply:
@hypest, this one was under the "I'll get to that later!" list in my mind that I hadn't got round to yet. I believe I was just exploring a few things to see if they worked when I placed the functions under `OnMediaLibraryButtonListener` and hadn't got round to implementing `OnSetFeaturedImageListener` yet. I've done that now. :) Thanks for the push!